acoustid/data/track.py

```
@@ -60,11 +57,14 @@
-    query = sql.select(
-        [schema.track_meta.c.track_id, schema.track_meta.c.meta_id],
-        sql.and_(schema.track_meta.c.track_id.in_(track_ids)),
-    ).order_by(
-        schema.track_meta.c.track_id, schema.track_meta.c.submission_count.desc()
+    query = (
+        sql.select(schema.track_meta.c.track_id, schema.track_meta.c.meta_id)
+        .where(
+            sql.and_(schema.track_meta.c.track_id.in_(track_ids)),
+        )
+        .order_by(
+            schema.track_meta.c.track_id, schema.track_meta.c.submission_count.desc()
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)